apps/settings/test/unit/panels/screen_lock_passcode/screen_lock_passcode_test.js

```diff
@@ -16,7 +17,8 @@ suite('ScreenLockPasscode > ', function() {
       'shared_mocks/mock_navigator_moz_settings',
       'shared_mocks/mock_settings_listener',
       'unit/mock_settings_service',
-      'panels/screen_lock_passcode/screen_lock_passcode'
+      'panels/screen_lock_passcode/screen_lock_passcode',
+      'shared/passcode_helper'
```

Comment:
Defining a mock is a little bit tricky but you could refer to this test[[1]]. It includes three steps:
1. Set the map for replacing a certain module. [[2]]
2. Make requirejs create a new context, so that the mock can be defined under the context. [[3]]
3. Define the mock. Note that the name should be the same as the one specified in the map. [[4]]

Note that the context you get in step two should continue to be used to require other module because `testRequire` creates another context where the mock you define previously does not exist there.

[1]: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/test/unit/modules/apn/apn_list_test.js
[2]: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/test/unit/modules/apn/apn_list_test.js#L7
[3]: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/test/unit/modules/apn/apn_list_test.js#L21
[4]: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/test/unit/modules/apn/apn_list_test.js#L36